subsys/bluetooth/audio/tbs.c

```diff
@@ -1085,6 +1085,9 @@ static int originate_call(struct tbs_service_inst *inst,
 	BT_TBS_CALL_FLAG_SET_OUTGOING(call->flags);
 
 	hold_other_calls(inst, 1, &call->index);
+
+	notify_calls(inst);
```

Comment:
is it necessary to also notify before setting the new call to 'alerting'?

Reply:
Yes. 

The current implementation of TBS does not really keep a call in the dialing state, but the call state machine is actually
idle -> dialing -> alerting -> ...

So we should always notifying the `dialing` state and then when switching (immediately) to the `alerting` state we should notify again.